spec/lib/mailchimp_spec.rb

```diff
@@ -74,13 +77,29 @@
 																body: an_instance_of(String)
 														},
 														{
-																method: 'DELETE',
-																path: 'lists/list_id/members/on_mailchimp'
-														}
+															method: 'DELETE',
+															path: 'lists/list_id/members/on_mailchimp'
+														}								
 													])
 		end
   end
 
+	describe '.create_nonprofit_user_subscribe_body' do 
+		let(:nonprofit) { create(:nonprofit)}
+
+		it 'creates nonprofit user subscriber' do 
+			expect(Mailchimp.create_nonprofit_user_subscribe_body(nonprofit, user)).to match({
+				'email_address' => user.email,
+				'status' => 'subscribed',
+				'merge_fields' => {
+					'NONPROFIT_ID' => nonprofit.id
+				}
+			})
+		end 
+
+	end 
```

Comment:
Were we previously NOT testing this function before this review?

Reply:
Yes, however, that was because the ticket was out of order. 
Also, the test cases that I had at that time did not run.